### PR TITLE
Refactored the cities module from charts package

### LINKED ISSUE #478: Review the English translations for the site
# Objetivos / Objectivos / Purpose

## Geral / General

Review translations pages:
- [x] Inicio
- [x] Equipe
- [x] Saiba mais
- [x] Participe
- [x] Dados
  * [x] Api
  * [x] Tutorial
- [x] Relatórios
  * [x] Municipal
  * [x] Estadual
- [x] Alerta Municipal * https://info.dengue.mat.br/alerta/4122008/dengue
- [x] Alerta Estadual* https://info.dengue.mat.br/alerta/SP/dengue

### LINKED ISSUE #480: Translate the report of cities page to English
# Objetivos / Objectivos / Purpose

## Geral / General

- Reference #478

### LINKED ISSUE #492: Sem dados para mostrar nas visualizações
# Objetivos / Objectivos / Purpose

##### - Sem dados para mostrar nas visualizações:

Segue o resumo dos municípios que estão sem acesso aos dados em seus relatórios ou que acenderam para Zk e Chk:
``` 
Sem dados de DG, ZK e CHK: 
- Pedra Branca/CE
- Chapecó/SC
- Rio Branco/AC
- Bauru/SP
- Sorocaba/SP

Município de Roraima que acendeu para Zika (SE 49/50): São João da Baliza
- na SE 25 foram 26 notificações

Município de Rondônia que acendeu para Chk: Presidente Médici
- quando acessei a página principal consegui filtrar para casos de Chk, mas quando abri o relatório os dados de DG, ZK e CHK não estão disponíveis

Municípios do Tocantins:
Acesos para CHK e ZK
- Colinas do Tocantins: os dados de DG, ZK e CHK não estão disponíveis
- Palmas: 
Chk: notificações nas últimas 12 semanas;  amarelo para Chk desde a SE 43; SE 49: 20 notificações
Zk: sem dados

ZK
- Araguaína: sem dados 
``` 
O estado tem clima favorável desde a SE 33/2021
by @RibackThais

### PR DESCRIPTION
**Pull request by esloch**
- Closes #492.
- Closes #480.
- Closes #478.

### CODE REVIEW

File: AlertaDengue/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -42,54 +42,45 @@ msgstr "Last uploaded files:"
...
 
-#: AlertaDengue/dados/charts/cities.py:630
+#: AlertaDengue/dados/charts/alerts.py:145
 msgid "Alerta Verde de "
 msgstr "Green Level "
 
-#: AlertaDengue/dados/charts/cities.py:648
+#: AlertaDengue/dados/charts/alerts.py:163
 msgid "Alerta Amarelo de "
 msgstr "Yellow Level "
```
Comment (fccoelho) [inline AlertaDengue/locale/en/LC_MESSAGES/django.po:64]: same here.

File: AlertaDengue/locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -42,54 +42,45 @@ msgstr "Last uploaded files:"
...
 
-#: AlertaDengue/dados/charts/cities.py:666
+#: AlertaDengue/dados/charts/alerts.py:181
 msgid "Alerta Laranja de "
 msgstr "Orange Level "
```
Comment (fccoelho) [inline AlertaDengue/locale/en/LC_MESSAGES/django.po:68]: same

File: AlertaDengue/locale/en/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -42,54 +42,45 @@ msgstr "Last uploaded files:"
...
 
-#: AlertaDengue/dados/charts/cities.py:684
+#: AlertaDengue/dados/charts/alerts.py:199
 msgid "Alerta Vermelho de "
 msgstr "Red Level "
```
Comment (fccoelho) [inline AlertaDengue/locale/en/LC_MESSAGES/django.po:72]: same

File: AlertaDengue/locale/en/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -42,54 +42,45 @@ msgstr "Last uploaded files:"
...
 
-#: AlertaDengue/dados/charts/cities.py:703
+#: AlertaDengue/dados/charts/alerts.py:218
 msgid "Casos Estimados de "
 msgstr "Estimated cases "
```
Comment (fccoelho) [inline AlertaDengue/locale/en/LC_MESSAGES/django.po:76]: should be "Estimated cases of "

File: AlertaDengue/locale/en/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -1308,10 +1412,358 @@ msgstr "Report"
 msgid "Municipal"
 msgstr "By City"
 
-#: AlertaDengue/dados/templates/report_city.html:61
+#: AlertaDengue/dados/templates/report_city.html:60
 msgid "Relatório da Situação das Arboviroses em "
 msgstr "Arborivirosis Situation report of "
```
Comment (fccoelho) [inline AlertaDengue/locale/en/LC_MESSAGES/django.po:1417]: "Arbovirosis"

File: AlertaDengue/locale/en/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -1308,10 +1412,358 @@ msgstr "Report"
...
+msgid ""
+"A figura abaixo mostra o indicador meteorológico de receptividade do\n"
+"      município para a transmissão de arboviroses. Quando o indicador está\n"
+"      acima do limiar (linha horizontal), existem condições ambientais\n"
+"      favoraveis à transmissão viral. Isso significa que a atenção precisa "
+"ser\n"
+"      aumentada para a possivel circulação sustentada de arbovirus e para a\n"
+"      implementação de controle vetorial"
+msgstr ""
+"The figure below shows the reception weather indicator of the\n"
```
Comment (fccoelho) [inline AlertaDengue/locale/en/LC_MESSAGES/django.po:1442]: "...weather receptiveness indicator..."

File: AlertaDengue/locale/en/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -1308,10 +1412,358 @@ msgstr "Report"
...
+"    esse indicador é mais sensivel para detectar dengue do que outras\n"
+"    arboviroses"
+msgstr ""
+"The Figure below shows the time series of notified cases of \n"
+"arboviruses in the last week. The mention curve refers to \n"
+"intensity of mention of symptoms of arboviruses and presence of \n"
+"mosquitoes according to the population of the municipality using \n"
+"social networks. This indicator, which is most effective in cities \n"
+"larger, should be seen as an aid in the early detection of \n"
+"increase in cases. Currently, this indicator is more sensitivefor\n"
```
Comment (fccoelho) [inline AlertaDengue/locale/en/LC_MESSAGES/django.po:1472]: "... sensitive to the detection of dengue..."

File: AlertaDengue/locale/en/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -1308,10 +1412,358 @@ msgstr "Report"
...
+msgstr "Credits"
+
+#: AlertaDengue/dados/templates/report_city.html:319
+#: AlertaDengue/dados/templates/report_state.html:467
+msgid ""
+"Este foi um projeto desenvolvido com apoio da SVS/MS e é operado como "
+"parceira entre"
+msgstr ""
+"This was a project developed with support from SVS/MS and is operated "
+"aspartner between"
```
Comment (fccoelho) [inline AlertaDengue/locale/en/LC_MESSAGES/django.po:1734]: "... as a partnership between"

**Review by fccoelho — CHANGES_REQUESTED** [review]

**Comment by esloch** [comment]
Thanks for the review @fccoelho.

**Pull request merged**